contribution_stats.py

```
@@ -0,0 +1,74 @@
+        return False
+
+
+def should_skip_file(path):
+    base = os.path.basename(path)
+    # Skip dotfiles and dotdirs
+    if base.startswith("."):
+        return True
+    # Skip config files by extension
+    if base.lower().endswith(CONFIG_FILE_EXTENSIONS):
+        return True
+    return False
```

Ruff (0.12.2)

Return the condition directly

Inline condition

(SIM103)